Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -756,6 +757,13 @@ private void OnPlaybackStart(User user, BaseItem item)
                 data.Played = false;
             }
 
+            // When we start playing an AudioBookFile, mark all previous chapters as played, and
```

Comment:
We don't really do this for tv. I don't know if that is comparable though

Reply:
Right. This is the behavior I think most people would be expecting from audiobook playback. It makes it easier to resume from your last location (if you want to skip ahead or start from somewhere in the middle of a book started elsewhere, etc.) and ensures "Continue Listening" has a single, intuitive point (index and ticks).